[2025-10-16T02:41:36.933+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T02:41:36.949+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_transform.run_sql manual__2025-10-16T02:41:36.089368+00:00 [queued]>
[2025-10-16T02:41:36.953+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_transform.run_sql manual__2025-10-16T02:41:36.089368+00:00 [queued]>
[2025-10-16T02:41:36.953+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T02:41:36.958+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_sql> on 2025-10-16 02:41:36.089368+00:00
[2025-10-16T02:41:36.967+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_transform', 'run_sql', 'manual__2025-10-16T02:41:36.089368+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/silver_transform_dag.py', '--cfg-path', '/tmp/tmpjyek_t5w']
[2025-10-16T02:41:36.968+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask run_sql
[2025-10-16T02:41:36.966+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=225) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-16T02:41:36.970+0000] {standard_task_runner.py:63} INFO - Started process 228 to run task
[2025-10-16T02:41:36.979+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-16T02:41:36.999+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_transform.run_sql manual__2025-10-16T02:41:36.089368+00:00 [running]> on host 728de341d734
[2025-10-16T02:41:37.165+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_transform' AIRFLOW_CTX_TASK_ID='run_sql' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T02:41:36.089368+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-16T02:41:36.089368+00:00'
[2025-10-16T02:41:37.166+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T02:41:37.173+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.1, Python Version: 3.12.4, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-16T02:41:37.174+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-16T02:41:37.928+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:38.054+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:38.158+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:39.103+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:40.074+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:40.828+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:41.143+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-16T02:41:41.144+0000] {connection.py:762} INFO - closed
[2025-10-16T02:41:41.220+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2025-10-16T02:41:41.307+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-16T02:41:41.309+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T02:41:41.324+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=silver_transform, task_id=run_sql, run_id=manual__2025-10-16T02:41:36.089368+00:00, execution_date=20251016T024136, start_date=20251016T024136, end_date=20251016T024141
[2025-10-16T02:41:41.378+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T02:41:41.393+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-16T02:41:41.394+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
